.github/workflows/pre-commit.yaml

```diff
@@ -0,0 +1,28 @@
+name: pre-commit
```

Comment:
Let's swap this for the primary one from https://github.com/katapultmedia/.github.

Reply:
What do you mean swap?   Swap files?

Reply:
Use the setup workflow to use the primary one from KM.

Reply:
But I see it just largely copies it. :-D Let's roll on until we learn a bit more. lol